homework/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" type="text/css" href="main.css">
+  <title>Javascript3</title>
+</head>
+
+<body>
+  <div>
+    <header class="header">
+      <div class="container">
+        <h1>HYF Repositories</h1>
+        <select id="repositories" class="repo-selector">
+          <option>Please choose a option </option>
+        </select>
+      </div>
+    </header>
+    <div id="main" class="container">
+
```

Comment:
Please clean up all these excess blank lines; in your html as well as in your js and css :). You should only have blank lines to separate logical blocks.

Tip: you can use 'select all' to first remove *all* these blank lines and then add them back where they make sense.